.changeset/red-schools-complain.md

```diff
@@ -0,0 +1,5 @@
+---
+"@crowdstrike/tailwind-toucan-base": major
+---
+
+Update new Charts color palette for better accessibility and contrast
```

Comment:
**nit**

Could expand this slightly so that consumers have a bit better idea of what the major, breaking change is:

```suggestion
This release contains a breaking change by removing `surface-dash-widget` classes.  If you were previously using these classes, there is no replacement. Updated the existing Charts color palette for better accessibility and contrast.
```

Reply:
Thanks for calling that out - I didn't think to update the changeset message too!

I split this into two separate changeset messages, since that [seems to be supported](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md#you-can-add-more-than-one-changeset-to-a-pull-request) in a single PR.

Reply:
Excellent idea! I hadn't thought of that route.  Thanks a bunch 🎉